examples/results/predictions_with_recent_pv_20240704_163005.csv

```diff
@@ -0,0 +1,193 @@
+,power_kw,power_kw_no_live_pv
```

Comment:
As per my comment via slack I would remove this file for now till we have more results